resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -13,29 +13,28 @@
 			"m_c and r_c notice a cloud is shaped like a bird.",
 			"m_c points out a constellation to r_c.",
 			"m_c and r_c keep trying to figure out what a bad smell in camp is.",
-			"m_c poses a crazy theory to r_c and they chat about it for a bit.",
-			"The whole Clan sees m_c and r_c accidentally headbutt each other. Embarrassing.",

```

Comment:
Looks like a few events in this file got removed, I'll mark them for you.

Reply:
Most events that got removed in this "neutral" file were because I wasn't a big fan of them whether it's because they didn't read very smoothly or just didn't really make sense for a neutral interaction!